src/deadline/maya_submitter/job_bundle_output_test_runner.py

```diff
@@ -176,9 +166,6 @@ def run_maya_render_submitter_job_bundle_output_test():
                 # skip renderman tests if rfm is not available
                 if "renderman" in dcc_scene_file:
                     if not maya.cmds.pluginInfo("RenderMan_for_Maya.py", query=True, loaded=True):
-                        report_fh.write(
-                            f"Skipping test {test_name} because Renderman for Maya is not installed."
-                        )
```

Comment:
This would be good to put back as well, so that we can see when this test is skipped from log output.